### ISSUE TITLE
Crouton not installing

### ISSUE THREAD

**Issue by Faliureatlife** [dnschneid/crouton#3724]
```
chronos@localhost / $ sudo sh ~/Downloads/crouton -e -t xfce
Downloading latest crouton installer...
################################################################################################################################################ 100.0%################################################################################################################################################ 100.0%
/usr/local/chroots/xenial already has stuff in it!
Either delete it, specify a different name (-n), or specify -u to update it.```



I try to download crouton and that error message pops up

**Comment by DennisLfromGA** [comment]
@Faliureatlife,

When you use the installation command without specifying a release it will default to 'xenial'.
That's why you're getting the message: 
> /usr/local/chroots/**xenial** *already has stuff in it!*
Either delete it, specify a different name (-n), or specify -u to update it.

So just follow those instructions or specify a different release.
Most likely, since you already have a chroot, you can just update using:
- `sudo sh ~/Downloads/crouton -n xenial -u`

If you want to add the 'xfce' target to your existing chroot just use:
- `sudo sh ~/Downloads/crouton -n xenial -t xfce -u`

*Hope this helps,*
-**DennisLfromGA**

**Comment by Faliureatlife** [comment]
thank you

**Comment by Blueman411** [comment]
It says:
`sh: 0: Refusing to exec /home/chronos/user/Downloads/crouton from noexec mount; see https://chromium.googlesource.com/chromiumos/docs/+/master/security/noexec_shell_scripts.md`

What do I do?

**Comment by MrHallway0** [comment]
@Blueman411 this is addressed in #4026 and this issue is very old, in the future please look for the most up to date documentation.

**Comment by Blueman411** [comment]
oh ok thank you!
